.travis.yml

```diff
@@ -4,6 +4,7 @@ os:
     - osx
 node_js:
     - "4"
+    - "6"
```

Comment:
Why are we now testing on node 6? Aside from regressions in node, there will be nothing we don't pick up with 4 and 7